src/services/Api.ts

```diff
@@ -31,3 +31,7 @@ const newRequest = (method: string, url: string, queryParams: any, data: any) =>
 export const GetServiceMetrics = (namespace: String, service: String, params: any) => {
   return newRequest('get', `/api/namespaces/${namespace}/services/${service}/metrics`, params, {});
 };
+
+export const GetGraphElements = (namespace: String, params: any) => {
+  return newRequest('get', `/api/namespaces/${namespace}/graphs`, params, {});
```

Comment:
I have some concerns about this kind of REST api. What I mean - graphs is not a resource on a namespace. Users calling that kind of api do not receive any resource, they receive a specific representation of the entities/resources in that namespace. that should be an input parameter on a GET request on a namespace resource, not a separate api.

Reply:
Can you please open a JIRA for this so that we can keep track and discuss it there?
Its outside of the scope for this PR.